CODEOWNERS

```diff
@@ -1 +1 @@
-*    @giantswarm/team-batman
+*    @giantswarm/team-batman-engineers
```

Comment:
Changing this to engineers so we don't spam @cokiengchiara with review requests :)